README.md

```diff
@@ -84,4 +84,18 @@ return [
 ];
 ```
 
-You must make some changes to your OS, see this example: https://github.com/aik099/PhpStormProtocol
+You must make some changes to your OS. See these examples: 
+ - PHPStorm: https://github.com/aik099/PhpStormProtocol
+ - Sublime Text 3 on Windows or Linux: https://packagecontrol.io/packages/subl%20protocol
+ - Sublime Text 3 on Mac: https://github.com/inopinatus/sublime_url
+
+#### Virtualized or dockerized
+
+If your application is run under a virtualized or dockerized environment, it is often the case that the application's base path is different inside of the virtual machine or container than on your host machine. For the links work in those situations, you can configure `traceLine` like this (change the path to your app):
+
+```php
+'traceLine' => function($options, $panel) {
+    $filePath = str_replace(Yii::$app->basePath, '~/path/to/your/app', $options['file']);
+    return strtr('<a href="phpstorm://open?url={file}&line={line}">{file}:{line}</a>', ['{file}' => $filePath]);
```

Comment:
replace as in `DEFAULT_IDE_TRACELINE` `<a href="ide://open?url=file://{file}&line={line}">{text}</a>`